concert/handler.py

```diff
@@ -0,0 +1,88 @@
+"""
+handler.py
+----------
+Defines a remote logging handler which should facilitate logging at the remote
+host at a specified location.
+"""
+import logging
+import asyncio
+from typing import List, Protocol
+from concert.typing import RemoteDirectoryWalkerTangoDevice
```

Comment:
Why once "from typing" and once "from concert.typing". How about we make a package "loghandlers" and you put the typing and the implementation files there?